packages/eslint-plugin-pf-codemods/lib/rules/nav-list-variant.js

```diff
@@ -28,21 +35,35 @@ module.exports = {
             if (variantAttr.value !== null) {
               context.report({
                 node,
-                message: `variant has been removed from ${node.openingElement.name.name}, use <${navImportName} ${variantVal}> instead`,
+                messageId: "removeNavListVariant",
+                data: {
+                  navListName: node.openingElement.name.name,
+                  navName: navImportName,
+                  variantVal: variantVal
+                },
                 fix(fixer) {
-                  const fixes = [fixer.replaceText(variantAttr, '')];
+                  const fixes = [];
                   if (hasNavParent) {
+                    fixes.push(fixer.replaceText(variantAttr, ''));
                     fixes.push(fixer.insertTextAfter(node.parent.openingElement.name, ' ' + variantVal))
+                  } else {
+                    fixes.push(fixer.replaceText(variantAttr, `/*TODO: move to Nav - ${variantVal}*/`));
```

Comment:
before possible values for variant included: `'default' | 'simple' | 'horizontal' | 'tertiary'`
If the user had `default` or `simple` before we can just remove it, otherwise comment it out

Reply:
Done